README.md

```diff
@@ -11,8 +11,27 @@ The repository for the backend can be found [here](https://github.com/mozilla/fi
 
 ## Prerequisites
 
-- Node
-- [Yarn](https://www.npmjs.com/package/yarn)
+
+## [Node](https://nodejs.org/en/)
+
+* ## Install node:
+
+Binaries, installers, and source tarballs are available at
+<https://nodejs.org/en/download/>.
+
+#### Current and LTS Releases
```

Comment:
remove this section, or merge it with above.  Does the person install node from https://nodejs.org/en/download/ or from https://nodejs.org/download/release ?

Reply:
removed this section